src/defs.h

```diff
@@ -154,8 +156,14 @@ const uint32_t TEST_FIRST_CONNECTION_FIRST_PACKET_TTL_THRESHOLD_MSEC = 50;
     "([a-zA-Z0-9\\.\\-]+|\\[([a-fA-F0-9.:]+(%[0-9a-zA-z])?)\\])"                                   \
     ":(6553[0-5]|655[0-2][0-9]|65[0-4][0-9]{2}|6[0-4][0-9]{3}|[0-5]?[0-9]{1,4})"                   \
     "(:([a-zA-Z0-9\\.\\-]+|\\[([a-fA-F0-9.:]+)\\]))?[\r\n]"
+
+#define UNIX_DOMAIN_SOCKET_FORMAT_REG_EXP                                                          \
+        "^[UuTt]:[\\/]([0-9a-zA-z\\/]+)[\r\n]"
```

Comment:
```suggestion
        "^[UuTt]:(/.+)[\r\n]"
```
Are you sure that `/` really needs escaping?